department-of-veterans-affairs/va.gov-cms
# VACMS-3603: Remove dupes from tag view.

## Description
See #3603 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/101488787-0a5bcf80-392e-11eb-949a-833a70d5b6c6.png)
![image](https://user-images.githubusercontent.com/2404547/101488768-05971b80-392e-11eb-91f1-dde147ca0b45.png)

## QA steps
- As an administrator, go to node/6924/edit and select `Service members` from beneficiares vocab.
- Go to `/resources/tag/service-members` and visually verify that this node appears
- Go to to `resources/tag/sign-in` and verify that duplicates do not appear (note that some content will have the same title, so will be necessary to hover over edit link to confirm unique url)

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
